packages/integrations/node/src/serve-app.ts

```diff
@@ -25,6 +25,8 @@ export function createAppHandler(app: NodeApp): RequestHandler {
 		try {
 			request = NodeApp.createRequest(req);
 		} catch (err) {
+			logger.error(`Could not render ${req.url}`);
+			console.error(err);
```

Comment:
Why don't we use our logger to log the actual error?

Reply:
I've just used the same approach as [here](https://github.com/withastro/astro/blob/368e4ced10b7a629c9ad94800ccf2a1243038921/packages/integrations/node/src/middleware.ts#L33).  I'm happy to change it.